homeassistant/components/knx/schema.py

```diff
@@ -84,9 +86,10 @@ class BinarySensorSchema:
 
     CONF_STATE_ADDRESS = CONF_STATE_ADDRESS
     CONF_SYNC_STATE = CONF_SYNC_STATE
+    CONF_INVERT = CONF_INVERT
```

Comment:
```suggestion
```

Reply:
Same reasoning: https://github.com/home-assistant/core/pull/42026#discussion_r522011139 Still want us to change this? I'd prefer to have it there as it defines the complete schema

Reply:
It's fine.